### DISCUSSION TITLE
Markdown URLs within local directory not rendered correctly

### DISCUSSION THREAD

**Discussion by systolan** [requarks/wiki#3663] [Error / Bug Report]
Within a markdown document, residing in a sub-folder, an URL to another document within the same sub-folder over "./" is not rendered correctly on the resulting pages.

**Steps to reproduce**
1. create a markdown document inside a sub-folder
1. create a link in the form of `[link](./other-document.md)`
1. observe the link is correct (!) in the preview
1. save and close the document
1. observe the actual link is now **incorrect** (!) and points to the same document we just edited.

**Host Info :**
 - OS: Ubuntu Server 18.04
 - Wiki.js version: 2.2.51 dockerized as described on the docs
 - Database engine: postgres 11.7

**Comment by Smankusors** [comment]
This is similiar, or duplicate of #1547, #1201
